tests/resource_model_test.py

```diff
@@ -168,19 +161,6 @@ def test_nones_for_dates_on_model_types_creation(self):
         )
 
     @httpretty.activate
-    def test_success_on_model_types_instantiation(self):
```

Comment:
The test name might be misleading, but i think the test checks the `__eq__` method implementation of the model. Is the test redundant / unnecessary?